combine_masks.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python
+
+import sys
+import os
+import numpy as np
+import argparse
+import netCDF4 as nc
+
+def fix_mask(mask1_filename, mask2_filename, grid_filename, lat, output_mask_filename):
+
+    lat = -60 # we will fix the mask south of this latitude
```

Comment:
remove line (set `lat=-60` as default in arg list?)

Reply:
Oh, that should not be there (it was a leftover of an older version of the script). The corresponding parser argument is mandatory, so no need to set a default value. I've fixed this, which in turn uncovered another issue, as the parser argument needs to be of type "float".